backend/package-lock.json

```diff
@@ -0,0 +1,6853 @@
+{
```

Comment:
We don't want to git track the `package-lock.json` files so you can remove this from tracked files

```
git rm package-lock.json
```